faker/providers/lorem/__init__.py

```diff
@@ -6,9 +6,24 @@
 
 class Provider(BaseProvider):
     @classmethod
+    def characters(cls, max_nb=255, variable_case=True):
```

Comment:
I don't think `variable_case` is necessary`. You could achieve the same by simply calling `Provider.characters().lower()` (or `.upper()`, or `.title()`)